src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	public static final int INITIAL_RHYMER_INDICATOR = 1;
+	public static final int MAX_SIZE = 12;
+	public static final int IF_EMPTY = 11;
```

Comment:
Wrong name. This constant is related to MAX_SIZE. Changing only one leads to an error (inconsistency).